plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -98,12 +82,7 @@ export const Modal = ({ open = true, toggleModal }: SearchModalProps) => {
     >
       <DialogTitle>
         <Paper className={classes.container}>
-          <SearchBarBase
-            className={classes.input}
-            value={value}
-            onChange={handleQuery}
-            onClear={handleClear}
-          />
+          <SearchBar className={classes.input} debounceTime={500} />
```

Comment:
to my comment above, if we set a higher debounceTime as default, we don't need  the debounceTime={500} prop here